plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -0,0 +1,1142 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, Eike Frost <ei@kefro.st>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_realm
+
+short_description: Allows administration of Keycloak realms via Keycloak API
+
+
+description:
+    - This module allows the administration of Keycloak realms via the Keycloak REST API. It
+      requires access to the REST API via OpenID Connect; the user connecting and the client being
+      used must have the requisite access rights. In a default Keycloak installation, admin-cli
+      and an admin user would work, as would a separate client definition with the scope tailored
+      to your needs and a user having the expected roles.
+
+    - The names of module options are snake_cased versions of the camelCase ones found in the
+      Keycloak API and its documentation at U(http://www.keycloak.org/docs-api/3.4/rest-api/).
+      Aliases are provided (if an option has an alias, it is the camelCase'd version used in the
+      API).
+
+options:
+    state:
+        description:
+            - State of the realm once this module has run
+            - On C(present), the realm will be created (or updated if it exists already).
+            - On C(absent), the realm will be removed if it exists
+        choices: ['present', 'absent']
+        type: str
+        default: 'present'
+
+    name:
+        type: str
+        description:
+            - Name of the realm to be acted on. Either this or I(realm) is required. If both are
+              given, I(name) specifies the current name of the realm, while I(realm) specifies the
+              new name thereof.
+
+    realm:
+        type: str
+        description:
+            - Name of the realm. Either this or I(name) is required.
+
+    access_code_lifespan:
+        type: int
+        description:
+            - Client login timeout in seconds.
+        aliases:
+            - accessCodeLifespan
```

Comment:
```suggestion
```
I suggest choosing one (the best) name for every parameter and removing all aliases. Let's use one approach. Usually people add aliases when modules and options already exist (typically when a typo or not the best name was chosen).

Reply:
I agree, though the same thing is already done by at least one other keycloak module. Is there a good reason why the alias should be there? (I have no experience with keycloak, so I'd be happy if @toniiiik and/or @eikef could elaborate on this.)

Reply:
In that case, I think it's ok to have them. (Also because the other modules already have them.)